src/components/TextLink.js

```diff
@@ -32,20 +32,35 @@ const defaultProps = {
 
 const TextLink = (props) => {
     const additionalStyles = _.isArray(props.style) ? props.style : [props.style];
+
+    /**
+     * Checking Function on open TextLink identify whether Clicked or Entered key press
+     *
+     * @param {Event} event
+     * @returns {Function} Returns onPress function.
+     */
+
+    const openLink = (event) => {
+        if (event.key !== 'Enter' && event.type !== 'click') {
```

Comment:
Can you please add a code comment to explain why this logic is here? (I know why, but the next engineer that sees it will probably be confused).

Reply:
Actually, disregard this. I see this is covered in the method description.